src/test/java/com/synopsys/integration/detect/tool/detector/inspectors/nuget/DotNetRuntimeAvailabilityVerifierTest.java

```diff
@@ -0,0 +1,52 @@
+package com.synopsys.integration.detect.tool.detector.inspectors.nuget;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+import org.junit.Assert;
```

Comment:
You are using the wrong version of junit for some of your asserts.

Reply:
Good catch.